server/routes/unleashRoutes.ts

```diff
@@ -2,24 +2,34 @@ import unleashClient = require("unleash-client");
 
 const { initialize, Strategy } = unleashClient;
 
-class ByEnhetAndEnvironment extends Strategy {
+class ByDevEnhet extends Strategy {
   constructor() {
-    super("byEnhetAndEnvironment");
+    super("byDevEnhet");
   }
 
   isEnabled(parameters: any, context: any) {
-    if (process.env.NAIS_CONTEXT === "dev") {
-      return true;
-    }
     if (!context.valgtEnhet) {
       return false;
     }
 
     const valgtEnhetMatches =
       parameters.valgtEnhet.indexOf(context.valgtEnhet) !== -1;
-    const environmentEnabled = parameters.tilgjengeligIProd === "true";
 
-    return valgtEnhetMatches && environmentEnabled;
+    return valgtEnhetMatches && process.env.NAIS_CONTEXT === "dev";
+  }
+}
+
+class ByEnhet extends Strategy {
+  constructor() {
+    super("byEnhet");
+  }
+
+  isEnabled(parameters: any, context: any) {
+    if (!context.valgtEnhet) {
+      return false;
+    }
+
+    return parameters.valgtEnhet.indexOf(context.valgtEnhet) !== -1;
```

Comment:
Mulig jeg har misforstått, men hvis vi ikke tester for om miljøet er prod her, vil vi ikke gi tilgang til samme enhet i dev også? Det er jo ikke noe farlig i så fall, men kan kanskje være forvirrende?

Reply:
Det er sant, men jeg trodde kanskje det var meningen? Enten `byDevEnhet`, som gir enhetene tilgang i dev, eller `byEnhet`, som gir enhetene tilgang overalt. Er det noen caser der vi bare vil gi tilgang i prod og ikke i dev?